name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/StatementOfAssetsViewer.java

```diff
@@ -370,7 +370,28 @@ public String getText(Object e)
         column.setVisible(false);
         support.addColumn(column);
 
-        column = new Column("8", Messages.ColumnPurchaseValue, SWT.RIGHT, 80); //$NON-NLS-1$
```

Comment:
Die bisherigen Spalten wurden umbenannt ("9" zu "10", etc.), damit die internen Namen in ihrer Reihenfolge zusammenpassen mit ihrer Ordnung im Menü. Nachteil: wenn Portfolio-XMLs geöffnet werden, die von einer Portfolio-Version vor diesem Feature stammen, werden die falschen Spalten angezeigt.

Reply:
Die "Kennung" der Spalte darf sich nicht ändern, denn so wird die Tabellenkonfiguration lokal gespeichert. Das wäre eine inkompatible Änderung und zerstört die Tabellenkonfiguration bei den Benutzern.

Aus historischen Gründen haben einige Spalten noch Nummern (anfangs habe ich mir nur den Index gemerkt). Aber neue Spalten können einen beliebigen String verwenden - also sowas wie "mapp" für moving average purchase price.